modules/monitoring-granting-a-user-access-to-metrics.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * monitoring/user-workload-monitoring.adoc
+
+[id="granting-a-user-access-to-metrics_{context}"]
+= Granting a user access to metrics
+
+By default, only cluster administrator users have access to user workload metrics. Using the soft tenancy feature, you can also grant metrics access to an arbitrary user, who only has access to a particular project.
+
+.Prerequisites
+
+* You need to have deployed the workload that you want to monitor, such as an application or a service. This example uses the application described in xref:../monitoring/user-workload-monitoring.adoc#deploying-a-sample-application[Deploying a sample application].
+* You need to have user workload monitoring enabled.
+* You need to have a user created. For the sake of an example, you can create the `johnsmith` user by running `oc create user johnsmith`.
```

Comment:
the user is linked with IDENTITIES, the user should be exist in the IDP server, for example: googleidp
$ oc get user
NAME                UID                                    FULL NAME      IDENTITIES            
Mia@example.com   051c0a54-3ee9-4822-8c48-ad1711064c90   Example   googleidp:116782835371291756106

Reply:
I see. I deleted the command. Giving full instructions on creating a user is off-topic here.